plugins/devices/README.md

```diff
@@ -0,0 +1,235 @@
+# Devices: The Device Enablement Plugin
+
+The `devices` plugin installs the `benchmark` tool into the `turnkey` CLI, which in turn provides support for running models against target runtimes and hardware.
+
+The tools currently support the following combinations of runtimes and devices:
+
+<span id="devices-runtimes-table">
+
+| Device Type | Device arg | Runtime                                                                               | Runtime arg                      | Specific Devices                              |
+| ----------- | ---------- | ------------------------------------------------------------------------------------- | -------------------------------- | --------------------------------------------- |
+| Nvidia GPU  | nvidia     | TensorRT<sup>†</sup>                                                                  | trt                              | Any Nvidia GPU supported by TensorRT          |
+| x86 CPU     | x86        | ONNX Runtime<sup>‡</sup>, Pytorch Eager, Pytoch 2.x Compiled | ort, torch-eager, torch-compiled | Any Intel or AMD CPU supported by the runtime |
+
+</span>
+
+<sup>†</sup> Requires TensorRT >= 8.5.2  
+<sup>‡</sup> Requires ONNX Runtime >= 1.13.1  
+<sup>*</sup> Requires Pytorch >= 2.0.0  
+
+## Getting Started
+
+> Note: these steps assume that your current working directory of your terminal is at the repository root.
+
+To install, run:
+ 
+`pip install -e plugins/devices`
+
+To learn more about the tool, run:
+
+`turnkey benchmark -h`
+
+To use the tool, run:
+
+`turnkey -i models/transformers/bert.py discover export-pytorch benchmark --runtime ort --device x86`
+
+
+# Definitions
+
+This package uses the following definitions throughout.
```

Comment:
Our documentation is starting to get scattered.  Should we move the more generic part of this doc (such as definitions) into `turnkeymo/docs`? Any way of consolidating documentation a bit more would with tkml itself would be nice.

Reply:
I don't have a position on this. To some extent this documentation is deliberately hidden because we're not trying to advertise the benchmarking functionality any longer.